nest-city-account/src/nases/nases.service.ts

```diff
@@ -66,28 +61,33 @@ export class NasesService {
 
   async searchUpvsIdentitiesByUri(uris: string[]) {
     const jwt = this.createTechnicalAccountJwtToken()
-    const response = await axios
-      .post(
-        `${process.env.SLOVENSKO_SK_CONTAINER_URI}/api/iam/identities/search`,
-        JSON.stringify({ uris }),
+    const result = await this.clientsService.slovenskoSkApi
+      .apiIamIdentitiesSearchPost(
+        {
+          uris: uris,
```

Comment:
```suggestion
          uris,
```

shorthand is sufficient